Vic2ToHoI4Tests/TestFiles/Mappers/MergeRules/merge_nations.txt

```diff
@@ -0,0 +1,36 @@
+one = {
+	merge = yes
+	master = ONE
+	slave = ONA
+	slave = ONB
+}
+
+two = {
+	merge = yes
+	master = TWO
+	slave = TWA
+	slave = TWB
+}
+
+three = {
+	merge = no
+	master = THR
+	slave = THA
+	slave = THB
+}
+
+# For Vic2World tests, as we can only have one copy of this file
+test_merge =
+{
+	merge = yes
+	master = FOO
+	slave = BAR
+	slave = BAZ
+}
+
+non_merge =
+{
+	merge = yes
```

Comment:
this is non_merge?

Reply:
The rule is active but the countries don't exist (covering one of the corner cases in how the rules are applied)